### ISSUE TITLE
Add mybinder

### ISSUE THREAD

**Issue by cchwala** [pycomlink/pycomlink#63]
Add option to start pycomlink on https://mybinder.org/

**Comment by cchwala** [comment]
This is the contenct of a potential `environment.yml`

```
name: pycomlink-dev
channels:
  - conda-forge
dependencies:
 - numpy
 - scipy
 - pandas>=0.22
 - matplotlib
 - numba
 - folium
 - h5py
 - xarray
 - shapely
 - pyproj
 - tqdm
 - pykrige
 - future
 - pytest
```

**Comment by cchwala** [comment]
Binder needs to install `pycomlink` itself, so maybe a environment.yml like this could work
```
name: pycomlink-dev
channels:
  - conda-forge
dependencies:
 - numpy
 - scipy
 - pandas>=0.22
 - matplotlib
 - numba
 - folium
 - h5py
 - xarray
 - shapely
 - pyproj
 - tqdm
 - pykrige
 - future
 - pytest
 -pip:
   -git+https://github.com/jpolz/pycomlink.git
```

**Comment by cchwala** [comment]
closed in #64
